typescript/cli/src/submit/submit.ts

```diff
@@ -44,7 +44,7 @@ async function getSubmitter<TProtocol extends ProtocolType>(
         ...submitterMetadata,
       });
     case TxSubmitterType.GNOSIS_SAFE:
-      return new EV5GnosisSafeTxSubmitter(multiProvider, {
+      return EV5GnosisSafeTxSubmitter.create(multiProvider, {
```

Comment:
Why does this one use a factory-like instantiator while the others above don't?

Reply:
Need to do async calls to validate safes. iiuc, constructors can't be async